packages/thrift-client/src/main/connections/request.ts

```diff
@@ -20,7 +20,9 @@ export class RequestConnection<TClient> extends HttpConnection<TClient> {
       // Encoding needs to be explicitly set to null or the response body will be a string
       encoding: null,
       headers: {
-        Connection: 'keep-alive',
+        'host': this.hostName,
+        'connection': 'keep-alive',
```

Comment:
Connection already seems to be set by Request. I don't know about Axios.

Reply:
I see Connection was set in the old version of the code so it may have been that code or the Request type.